Main.java

```diff
@@ -3,119 +3,221 @@
+
+                System.out.printf("The area of the parallelogram with base %.2f and height %.2f is %.2f", pBase, pHeight, p.Area());
+
+                System.out.println("p.Area");
+                System.out.printf("The perimeter of the parallelogram with First Pair: %.2f and SecondPair: %.2f is %.2f", q.getLength(), q.getBreadth(), q.calculatePerimeter());
+                System.out.println("");
+                System.out.println("");
+                System.out.println("");
+
+                //this should be in the Trapezoid case
+                System.out.println("Calculation for a Trapezoid");
+                System.out.println("Input first side");
+                float a = scan.nextFloat();
+                System.out.println("Input second side");
+                float b = scan.nextFloat();
+                System.out.println("Input third side");
+                float c = scan.nextFloat();
+                System.out.println("Input fourth side");
+                float d = scan.nextFloat();
+                Trapezoid t = new Trapezoid(a, b, c, d);
+                System.out.printf("The perimeter of the Trapezoid with First Side: %.2f, Second Side: %.2f, Third Side: %.2f, and Fourth Side: %.2f is %.2f \n ", a, b, c, d, t.calculatePerimeter());
+
+
+//for rhombus
+        System.out.println("\n area and perimeter of a rhombus ");
+        System.out.println("\n  input side number");
+        float sides1 = scan.nextFloat();
+        System.out.println("\n  input p");
+        float P = scan.nextFloat();
+        System.out.println(" \n input q");
+        float Q = scan.nextFloat();
+        Rhombus w = new Rhombus(sides1, P, Q);
+
+        System.out.printf("The perimeter of the rhombus with sides1 %.2f  is %.2f", sides1, w.calculatePerimeter());
+        System.out.println("p.Area");
+        System.out.printf("The area of the parallelogram with First Pair: %.2f and SecondPair: %.2f is %.2f", P, Q, w.Area());
+
+
+//for square
+        System.out.println("\n area and perimeter of a square");
+        System.out.println("input first value ");
+        float aa = scan.nextFloat();
+        System.out.println("input second value");
+        float bb = scan.nextFloat();
+        square r = new square(aa, bb);
+        System.out.println("\n perimeter");
+        System.out.println( r.calculatePerimeter());
+        System.out.println("\n area");
+        System.out.println( r.Area());
+
+        // for rectangle
+        System.out.println("\n area and perimeter of a rectangle ");
+        System.out.println("input first value ");
+        double d1 = scan.nextDouble();
+        System.out.println("\n input second value ");
+        double d2 = scan.nextDouble();
+        Rectangle u = new Rectangle(d1, d2);
+        System.out.println("\n perimeter");
+        System.out.println(u.Perimeter());
+        System.out.println("\n area");
+        System.out.println(u.Area());
+    }
+        }
```

Comment:
You should have used the switch case to allow the user make a choice of the quadilateral it will like to calculate